channels/uk.m3u

```diff
@@ -455,6 +529,8 @@ https://csm-e-stv.tls1.yospace.com/csm/live/139900483.m3u8
 https://csm-e-stv.tls1.yospace.com/csm/live/181023311.m3u8
 #EXTINF:-1 tvg-id="SupertoonsTV.uk" tvg-name="Supertoons TV" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/35BpWJh.jpg" group-title="Kids",Supertoons TV (720p)
 https://kedoo-supertoonstv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="Syfy.uk" tvg-name="Syfy" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/CDeBomC.png" group-title="",SYFY
+http://stream.tvtap.live:8081/live/uk-syfy.stream/playlist.m3u8
```

Comment:
Originates from the US.

Reply:
exemple   ads  are intended for uk https://i.imgur.com/iP0Lq8w.png  so it get confusing where to put them

Reply:
If the link is for a uk specific channel it goes in uk.m3u not us.m3u as the channel would not be shown/streamed in the US.

Reply:
SyFy is a US-based channel, but according to https://en.wikipedia.org/wiki/Syfy_(British_and_Irish_TV_channel), this one is UK-based and a totally different Syfy.

Reply:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.